FAQ.md

```diff
@@ -11,7 +16,22 @@ $ sequelize model:create --name User --attributes name:string,state:boolean,birt
 sequelize db:migrate
 ```
 ## How can I make a migrations rollback?
-
 ```
 sequelize db:migrate:undo:all
 ```
+
+## How can I create a seeder?
+```
+sequelize seed:create --name seederName 
```

Comment:
Explanation for name will be great https://github.com/sequelize/cli/blob/61104be8e59c0a7ec4cd0517620836d45826e417/FAQ.md#how-can-i-generate-a-model